lib/plugins/input/gelf.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+var gelfserver = require('graygelf/server')
+var safeStringify = require('fast-safe-stringify')
+var consoleLogger = require('../../util/logger.js')
+
+
+function InputGELF (config, eventEmitter) {
+  this.config = config.configFile.input.gelf
+  this.config.port = config.configFile.input.gelf.port || 12100
```

Comment:
Note, we should not use `config.configFile.` - this property was only for some Logagent 1.0 modules, or for a special case where one plugin needs to know if another plugin is present.  Using config.configFile limits plugin to a single instance and forces user to use the name "gelf" to make the config working. 

Normally we should address plugin configuration properties directly via "config.property" e.g. "config.port". 
See e.g.  https://github.com/sematext/logagent-js/blob/master/lib/plugins/input/elasticsearchHttp.js#L8

Reply:
ok